tests/Blinky.cpp

```diff
@@ -1,27 +1,58 @@
 #include "BrainPad.h"
+#include <string>
 
```

Comment:
This file looks the same as samples/main.cpp. Pick one and get rid of the other unless there's a reason for both